### DISCUSSION TITLE
Preferred language track setting

### DISCUSSION THREAD

**Discussion by evofender** [chrisbenincasa/tunarr#1022] [Ideas]
It would be very useful to have an option that would allow users to select a preferred language for the stream. Lots of multi-language files will stream in a foreign language rather than english since there's no way to force it.

It'll force people to remux entire libraries without this option.

**Comment by chrisbenincasa** [comment]
We now have this setting at the global level. You can configure your audio language preferences in priority order. We plan to bring this to the channel, and ultimately the media item level soon.
